oil_lang/objects.py

```diff
@@ -9,9 +9,10 @@
 from core.util import log
 from oil_lang import regex_translate
 
-from typing import TYPE_CHECKING
+from typing import Union, TYPE_CHECKING, Type, List, Dict, Any, Optional
```

Comment:
The Travis build is failing because `Type` is not defined in `vendor/typing.py`.

https://travis-ci.org/oilshell/oil/builds/621465709?utm_source=github_status&utm_medium=notification

The reason we have `vendor/typing` is because we don't need to ship the whole `typing.py` file with the production build of Oil.  It has some other dependencies on the Python interpreter that are stripped down version doesn't have.

Reply:
Fixed by: 02b1a4fa1162d1a18007acb62d5ee5d05b114f3f